src/System.Net.Http.Listener/src/System/Net/CookieExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+
+namespace System.Net
+{
+    internal static class CookieExtensions
```

Comment:
Not as far as I could tell.  ToServerString needs to know whether various values were explicitly set or not, and this isn't exposed.  It *may* be possible to derive the "variant" info from the public properties, but I'm not sure it can be done with full fidelity.